### ISSUE TITLE
Consider supporting options for browser-level scaling in the Electron client to better assist users with visual impairment.

### ISSUE THREAD

**Issue by jacksands** [foundryvtt/foundryvtt#10469]
### User Experience

I just love foundry, but me and also a friend have a minor vision impairments, we make use of Browsers ZOOM feature to help, and it is quite easy and efficient.

Unfortunately, when GMing, we cannot make use of the same system because it don't exist in the Foundry in the same way, so we open the app and then use a Browser to GM, forcing us to have both app running simultaneously on the pc.

There is a font size options, but it doesn't change some of the bars, many sheets, modules windows, etc.

I know for a fact that some people even use Zoom OUT to reduce the interface for various reasons, so it would help more than people like me.

Since foundry runs in what seems a browser like app, can you please take in consideration applying this kind of zoom options, and if this can be on an easy access place, instead of burrowed in the setting would be marvelous!

Perhaps a dedicated icon to fast access accessibility options, I think it also would inspire developers to create more accessibility focused modules!

![foundry zoom](https://github.com/foundryvtt/foundryvtt/assets/7229143/f974432c-efd9-4d1a-b86c-08631a185c45)
Browser zoom is usually smother and affects everything.

![foundry zoom 2](https://github.com/foundryvtt/foundryvtt/assets/7229143/0f1e73a7-4d35-4173-8431-5c392766b13b)
foundry Font zoom do not apply over everything and is quite limited

In any way, thanks for all your efforts and for creating what gives me this amazing opportunity to have fun!

**Comment by unsoluble** [comment]
For what it's worth, I don't _think_ it's at all trivial to add UI zoom controls to the Electron client; you'll probably need to continue to use a standalone browser for this.

Note that the app only needs to be _launched_, not logged into a game — with it launched and minimized, it should have essentially zero impact on machine performance.

**Comment by aaclayton** [comment]
Closed as duplicate of https://github.com/foundryvtt/foundryvtt/issues/5508
